presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -339,6 +390,12 @@ public void acknowledgeResults(
         taskManager.acknowledgeTaskResults(taskId, bufferId, token);
     }
 
+    /**
+     * shuffle related
+     *
+     * The downstream task has the option to close a buffer in the upstream task if it does not require more data.
```

Comment:
perhaps. "Deletes all the results accumulated in the output buffer for the specified task. This may happen if downstream task doesn't need all the data to complete execution. For example, LIMIT operator can complete execution after it has seen the requested number of rows."